src/main/java/com/ays/location/model/entity/LocationEntity.java

```diff
@@ -0,0 +1,48 @@
+package com.ays.location.model.entity;
+
+import com.ays.common.model.entity.BaseEntity;
+import com.ays.user.model.entity.UserEntity;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Point;
+
+/**
+ * Location entity, which holds the information regarding location.
+ */
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "AYS_USER_LOCATION")
+public class LocationEntity extends BaseEntity {
+
+    @Id
+    @Column(name = "ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "USER_ID")
+    private String userId;
+
+    @Column(name = "POINT", columnDefinition = "geometry(Point,4326)")
```

Comment:
`columnDefinition` alanı Point objesi setlenirken kullanılıyorsa `ST_GeomFromText(Point, 4326)` şeklinde yazılması gerekiyor. Farklı bir şey için kullanılıyorsa bu tanımı yapmaya ihtiyaç kalmıyor veritabanı katmanında bu tanım yapılmış durumda. Burayı konuşup netleştirebiliriz

Reply:
Gerekli değişiklikleri yaptım. Kontrol edebilir misin? @agitrubard